self-hosted.md

```diff
@@ -0,0 +1,135 @@
+# Creating a Self-Hosted Cluster
+
+A self-hosted cluster is a cluster that acts as both the management cluster and the workload cluster. A self-hosted cluster manages its own lifecycle.
+
+In this this section we will create a workload cluster and make it self-hosted by converting it into its own management cluster.
+
+<!-- table of contens generated via: https://github.com/thlorenz/doctoc -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Creating a Self-Hosted Cluster](#creating-a-self-hosted-cluster)
+  - [Create a Cluster](#create-a-cluster)
+  - [Convert Workload Cluster to Management Cluster](#convert-workload-cluster-to-management-cluster)
+  - [Clean up](#clean-up)
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Create a Cluster
+
+Firstly, lets create a workload cluster called `docker-cluster-self-hosted`.
+
+These steps will similar to how we created `docker-cluster-one`.
+
+* Create the cluster
```

Comment:
```suggestion
Create the cluster
```
I think this would look better and more consistent with other places in the tutorial (similar below)